src/dataExplorer/components/FluxBrowser.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {FC, useCallback, useContext, useEffect, useMemo} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+// Components
+import SelectorTitle from 'src/dataExplorer/components/SelectorTitle'
+import FilterList from 'src/shared/components/FilterList/FilterList'
+import FluxDocsTooltipContent from 'src/shared/components/functions/perFunction/FluxDocsTooltipContent'
+import Fn from 'src/shared/components/FilterList/MinimalistInjectOption'
+
+// State
+import {getFluxPackages} from 'src/shared/actions/fluxDocs'
+import {getAllFluxFunctions} from 'src/shared/selectors/app'
+import {EditorContext} from 'src/shared/contexts/editor'
+
+// Types
+import {FluxFunction} from 'src/types/shared'
+
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+import {getFluxExample} from 'src/shared/utils/fluxExample'
+import {sortFuncs} from 'src/shared/components/functions/utils'
+import './FluxBrowser.scss'
+
+const TOOLTIP = `The flux standard library contains several packages, \
+functions, and variables which may be useful when constructing your flux query.`
+
+const FluxBrowser: FC = () => {
+  const {editor, injectFunction} = useContext(EditorContext)
+
+  const dispatch = useDispatch()
+  const fluxFunctions = useSelector(getAllFluxFunctions)
+
+  useEffect(() => {
+    if (fluxFunctions.length === 0) {
+      dispatch(getFluxPackages())
+    }
+  }, [])
+
+  const handleSelectItem = useCallback(
+    (func: FluxFunction) => {
+      injectFunction(getFluxExample(func), _ =>
+        console.warn('Determine use of query context.')
```

Comment:
This is where we normally do a callback to a QueryContext.
But with our new plan for how to inject code into the editor, it may be better to just call `editor.getModel().getValue()` as the src of truth.